docs/security-legal-pii/security/ip-ranges.mdx

```diff
@@ -127,29 +126,47 @@ These IP addresses are only for Sentry use.
 
 ## Uptime Monitoring
 
-Sentry uses the following IP addresses for uptime checks:
-
-US
+Sentry uses the following IP addresses for uptime checks for all regions:
 
 ```plaintext
 34.123.33.225
 34.41.121.171
 34.169.179.115
 35.237.134.233
 34.85.249.57
-```
-
-EU
-
-```plaintext
 34.159.197.47
 35.242.231.10
 34.107.93.3
 35.204.169.245
 ```
 
+### Dynamic IP Address Retrieval
+
+For the most up-to-date list of Uptime Monitoring IP addresses, you can query the following API endpoint:
+
+```plaintext
+https://us.sentry.io/api/0/uptime-ips/
+```
+
+This endpoint returns a newline-separated list of all current Uptime Monitoring IP addresses (for all regions) that can be programmatically processed in your allowlist configurations.
+
 <Alert title="Note" level="warning">
 
 Uptime Monitoring IP addresses may change over time. If you need to programmatically verify whether a visitor is a Sentry bot, we recommend checking the [user agent](/product/alerts/uptime-monitoring/troubleshooting/#user-agent) instead.
 
 </Alert>
+
+### Information for Hosting Providers
+
+If you're a hosting provider (like Vercel, Netlify, or similar platforms), please consider the following guidance regarding Sentry's Uptime Monitoring:
+
+Many organizations use Sentry to monitor the uptime of their websites and applications. When Sentry's monitoring IPs are blocked, it creates false outage alerts for all customers trying to monitor sites on your platform.
+
+**Recommended actions for hosting providers:**
+
+1. Allowlist Sentry's monitoring IP addresses in your platform's security rules
+2. Configure your firewall to recognize Sentry's monitoring user agent and IP ranges as legitimate traffic
```

Comment:
We probably don't want to recommend looking at the user agent -- since this can be spoofed